packages/donations-app/app/components/common/filters.js

```diff
@@ -77,7 +77,10 @@ angular.module('aliceApp')
   })
 	.filter('vowelize', function() {
 		return function(word) {
-			const vowelRegex = '^[aieouAIEOU].*';
+      const vowelRegex = '^[aieouAIEOU].*';
```

Comment:
Why do we always need to reinvent the wheel :) 
As far as I know it's a bit more tricky eg. a user not an user. 
Let's use an existing lib to handle that: 
https://www.npmjs.com/package/articles

Reply:
I didn't add this, just fixed error in console